elements/rh-cta/docs/00-overview.md

```diff
@@ -0,0 +1,228 @@
+times in the same container except for the Primary variants.
+
+{% example palette="lightest",
+           style="--inline-img-max-width: 552px;",
+           alt="Call to action component usage, hierarchy",
+           src="/elements/cta/cta-usage-hierarchy.svg" %}
+
+#### Navigation vs. action
+
+Even though they look similar, call to action links and buttons have important 
+differences. Call to action links are navigational elements whereas buttons 
+perform actions.
+
+{% alert title="Learn more" %}
+  Visit the [Button](https://ux.redhat.com/components/button/){target="_blank"} 
+  component page to learn more about how to use buttons.
+{% endalert %}
+
+
+- Call to action links are used in layouts (like the home page) whereas buttons 
+  are used in components (like a 
+  [Form](https://ux.redhat.com/components/form/){target="_blank"}).
+- Call to action link text labels are larger and thicker than button text 
+  labels.
+- Call to action links with containers include more padding than buttons with 
+  containers.
+- Call to action link colors have different meanings than button colors.
+
+
+## Content {style="margin-block:var(--rh-space-3xl) 0"}
+
+Call to action links are navigational elements, so any text needs to 
+**accurately communicate the destination**. Inconsistent experiences might lead 
+to distrust, lessening the satisfaction of a user during their journey. Call to 
+action link text labels should be **action-oriented yet straightforward** so a 
+user will feel comfortable making a selection.
+
+{% alert state="warning", title="Warning" %}
+  Call to action link text labels need to be written clearly in order to be 
+  understood, therefore write **no more than four or five words** per link.
+{% endalert %}
+
+{% example palette="lightest",
+           style="--inline-img-max-width: 706px;",
+           alt="Call to action component usage, content",
+           src="/elements/cta/cta-usage-content.svg" %}
+
+### Character count
+
+Call to action link text labels should have fewer characters in order for their 
+message to remain impactful for a user.
+
+{% alert state="warning", title="Warning" %}
+  Do not write long text labels because they might break to two lines on small 
+  screens or weaken an impactful message.
+{% endalert %}
+
+{% section %}
+| Style     | Maximum characters |
+| --------- | ------------------ |
+| Primary   | 30                 |
+| Secondary | 35                 |
+| Brick     | 25                 |
+| Default   | 45                 |
+{% endsection %}
+
+### Variant grouping
+
+Variants can be grouped together if there are different kinds of links that need 
+to be displayed in the same container.
+
+{% alert title="Helpful tip" %}
+  If there are more than three links that can be grouped together **in a grid 
+  instead**, use the Brick variants.
+{% endalert %}
+
+{% alert state="warning", title="Warning", style="margin-block:var(--rh-space-2xl);" %}
```

Comment:
this is a nested p error

here's the generated html:

```html
<p><rh-alert state="warning" style="margin-block:var(--rh-space-2xl);"></rh-alert></p>
  <h3 slot="header">Warning</h3>
<p>Do not group <strong>more than three</strong> Primary, Secondary, or Default variants
together otherwise a user might have trouble making a selection.</p>
```